src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/toolbars/CustomizedToolbarComponentObject.java

```diff
@@ -301,4 +303,15 @@ public void verifyToolInMoreTool(String toolName) {
     }
     PageObjectLogging.log("verifyToolInMoreTool", toolName + " appears in ToolbarMoreTool.", true);
   }
+
+  /**
+   * Get text string from Theme Designer button in My Tools menu
+   *
+   */
+
+  public String getThemeDesignerText()
+  {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this left curly brace to the end of previous line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ALeftCurlyBraceEndLineCheck)